x-pack/plugins/secops/public/store/local/hosts/selectors.ts

```diff
@@ -11,6 +11,11 @@ import { HostsModel } from './model';
 
 const hostsQuery = (state: State): HostsModel => state.local.hosts;
 
+export const authorizationsLimitSelector = createSelector(
```

Comment:
I don't think this selector is named properly, though I see it follows the convention defined in this file. Why?

The return value for this selector is:
```
hosts => hosts.query.authorizations
```

Today, this selector is being used to grab the `limit` property from `authorizations`, but tomorrow we could use the same selector to grab a `foo` property.

This, would `authorizationsSelector` be a better name?

Reply:
Yea, I see two thumbs ups and this is correct as discussed. I will with this PR just rename all of these to be called `xyzSelector`